src/components/dashboard/sidebar/tests/EnterpriseOffersSummaryCard.test.jsx

```diff
@@ -46,44 +43,25 @@ describe('<EnterpriseOffersSummaryCard />', () => {
     expect(screen.getByTestId('offer-summary-text')).toBeInTheDocument();
   });
 
-  it('should render detailed summary text if remainingBalanceForUser is not null', () => {
-    const offers = [
-      mockEnterpriseOfferMaxUserDiscount,
-      {
-        ...mockEnterpriseOfferMaxUserDiscount,
-        remainingBalanceForUser: 100,
-      },
-    ];
-    render(
-      <EnterpriseOffersSummaryCard
-        offers={offers}
-      />,
-    );
-    // calculate sum of balance available for user across all offers
-    const expectedMaxUserDiscountSum = offers.reduce((acc, offer) => {
-      if (offer.remainingBalanceForUser) {
-        return acc + offer.remainingBalanceForUser;
-      }
-      return acc;
-    }, 0);
-    expect(screen.getByTestId('offer-summary-text-detailed')).toBeInTheDocument();
-    expect(screen.getByText(`$${expectedMaxUserDiscountSum}`)).toBeInTheDocument();
-  });
-
-  it('should render earliest offer end date, if applicable', () => {
+  it('should render earliest, current offer end date, if applicable', () => {
     render(
       <EnterpriseOffersSummaryCard
         offers={[
           mockEnterpriseOffer,
           {
             ...mockEnterpriseOffer,
-            endDatetime: '2022-04-01T00:00:00Z', // earliest start date
+            isCurrent: false,
+            endDatetime: '2022-04-01T00:00:00Z', // earliest, non-current start date
+          },
+          {
+            ...mockEnterpriseOffer,
+            endDatetime: '2023-12-01T00:00:00Z', // earliest, current start date
           },
         ]}
       />,
     );
 
     expect(screen.getByTestId('offer-summary-end-date-text')).toBeInTheDocument();
-    expect(screen.getByText('2022', { exact: false })).toBeInTheDocument();
+    expect(screen.getByText('2023', { exact: false })).toBeInTheDocument();
```

Comment:
Tiny suggestion: maybe make this text slightly more specific, like "2023-12-01", just to be slightly defensive about changes we might make to it in the future (like if someone changes 2022-04-01 to 2023-04-01).